tests/test_acquisition.py

```diff
@@ -246,6 +308,107 @@ def test_tzinfo(self):
             self.assertTrue(acq.acquisition_datetime, None)
 
 
+class Landsat7MtlRTC2AcquisitionTest(unittest.TestCase):
+    def setUp(self):
+        self.acqs = acquisitions(LS7_SCENERTC2).get_acquisitions(group="RES-GROUP-1")
+
+    def test_type(self):
+        for acq in self.acqs:
+            self.assertTrue(isinstance(acq, LandsatAcquisition))
+
+    def test_band_type(self):
+        self.assertEqual(self.acqs[0].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[1].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[2].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[3].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[4].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[5].band_type, BandType.THERMAL)
+        self.assertEqual(self.acqs[6].band_type, BandType.THERMAL)
+        self.assertEqual(self.acqs[7].band_type, BandType.REFLECTIVE)
+
+    def test_acquisition_datetime(self):
+        for acq in self.acqs:
+            self.assertEqual(
+                acq.acquisition_datetime,
+                datetime.datetime(2021, 2, 20, 1, 32, 16, 844238),
+            )
+
+    def test_min_radiance_band1(self):
+        self.assertEqual(self.acqs[0].min_radiance, -6.2)
+
+    def test_max_radiance_band1(self):
+        self.assertEqual(self.acqs[0].max_radiance, 191.6)
+
+    def test_min_quantize_band1(self):
+        self.assertEqual(self.acqs[0].min_quantize, 1.0)
+
+    def test_max_quantize_band1(self):
+        self.assertEqual(self.acqs[0].max_quantize, 255.0)
+
+    def test_solar_azimuth(self):
+        self.assertEqual(self.acqs[0].solar_azimuth, 75.86380542)
+
+    def test_solar_elevation(self):
+        self.assertEqual(self.acqs[0].solar_elevation, 42.86386904)
+
+    def test_gain(self):
+        self.assertAlmostEqual(self.acqs[0].gain, 0.77874, 4)
+
+    def test_bias(self):
+        self.assertAlmostEqual(self.acqs[0].bias, -6.97874, 4)
+
+    def test_sensor_id(self):
+        for acq in self.acqs:
+            self.assertEqual(acq.sensor_id, "ETM+")
+
+    def test_platform_id(self):
+        for acq in self.acqs:
+            self.assertEqual(acq.platform_id, "LANDSAT_7")
+
+    def test_samples(self):
+        self.assertEqual(self.acqs[0].samples, 75)
+
+    def test_lines(self):
+        self.assertEqual(self.acqs[0].lines, 65)
+
+    def test_read(self):
+        self.assertEqual(self.acqs[0].data()[40, 30], 50)
+
+    def test_spectral_filter_cfg_vsir(self):
+        self.assertEqual(self.acqs[0].spectral_filter_name, "landsat7_vsir.flt")
+
+    def test_spectral_filter_cfg_thermal(self):
+        self.assertEqual(self.acqs[5].spectral_filter_name, "landsat7_thermal.flt")
+
+    def test_temperature61(self):
```

Comment:
Does `61` have some contextual meaning? (and 62 below?)

Reply:
Good question. Not that I'm aware of.  This test is based on the Landsat7 Collection 1 test.

Reply:
So 61 and 62 are the band labels from USGS for Landsat-7.